downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -235,6 +235,11 @@ abstract class VersionControlSystem {
      */
     protected abstract val movingRevisionNames: List<String>
 
+    /**
+     * A list of lowercase names that clearly identify the VCS. For example ("svn", "subversion") for Subversion.
+     */
+    protected abstract val names: List<String>
```

Comment:
`names` is ok, just thinking if `aliases` would be better. I'll leave this to you.